Description: Revert "Extra configure check for libmp4v2 compatibili ty"
 This reverts upstream commit bed8f03202336fc7863ec6a39e130473f59061c6.
 "We do not support the newer libmpv2 version yet, so don't error out when
    compiling, catch it in configure."
 .
 Patch 0006a_fix_libmp4v2_build.patch adds supports for newer libmp4v2 and this
 commit just adds an error.
Author: Miguel A. Colón Vélez <debian.micove@gmail.com>
Forwarded: not-needed
Last-Update: 2014-11-30

--- a/acinclude.m4
+++ b/acinclude.m4
@@ -533,8 +533,7 @@
         AC_MSG_ERROR([unable to configure $1 support])
     fi
 
-    if ((test "x$mt_[]translit($1, `/.-', `___')_status" = xyes) &&
-        (test "x$9" != "xundef")); then
+    if test "x$mt_[]translit($1, `/.-', `___')_status" = xyes; then
         AC_DEFINE(translit(HAVE_$1, `a-z/.-', `A-Z___'), [1], [$1 library presence])
     fi
     
--- a/configure.ac
+++ b/configure.ac
@@ -1428,7 +1428,7 @@
 
 MT_CHECK_OPTIONAL_PACKAGE([libmp4v2], [disable],
                  [libmp4v2 support for mp4 metadata extraction],
-                 [$MP4_HEADER], [mp4v2], [MP4Read], [], [], [undef]) 
+                 [$MP4_HEADER], [mp4v2], [MP4Read]) 
 
 if test "x$LIBMP4V2_STATUS" = xyes; then
    CFLAGS="$CFLAGS $LIBMP4V2_CFLAGS"
@@ -1437,37 +1437,6 @@
    AC_DEFINE_UNQUOTED([LIBMP4V2_INCLUDE], [<$MP4_HEADER.h>], [$MP4_HEADER.h])
    AC_LANG_SAVE
    AC_LANG_CPLUSPLUS
-   AC_MSG_CHECKING([for libmp4v2 compatibility])
-   AC_COMPILE_IFELSE(
-        [AC_LANG_PROGRAM(
-            [
-                #include <$MP4_HEADER.h>
-            ],
-            [
-                (void)MP4GetMetadataName(0, 0);
-            ]
-        )],
-        [
-            AC_MSG_RESULT([yes])
-            AC_DEFINE([HAVE_LIBMP4V2], [1], [libmp4v2 library presence])
-        ],
-        [
-            AC_MSG_RESULT([not compatible])
-            if test "x$LIBMP4V2_OPTION_REQUESTED" = "xyes"; then
-                AC_MSG_ERROR([Unable to configure libmp4v2 support])
-            else
-                LIBMP4V2_STATUS="not compatible"
-            fi
-        ])
-    LDFLAGS="$LDFLAGS_SAVE"
-    CFLAGS="$CFLAGS_SAVE"
-    CXXFLAGS="$CXXFLAGS_SAVE"
-fi
-
-if test "x$LIBMP4V2_STATUS" = "xyes"; then
-   CFLAGS="$CFLAGS $LIBMP4V2_CFLAGS"
-   CXXFLAGS="$CXXFLAGS $LIBMP4V2_CFLAGS"
-   LDFLAGS="$LDFLAGS $LIBMP4V2_LDFLAGS $LIBMP4V2_LIBS"
    AC_MSG_CHECKING([for MP4GetMetadataCoverArtCount() in libmp4v2])
    AC_COMPILE_IFELSE(
         [AC_LANG_PROGRAM(
@@ -1506,10 +1475,7 @@
             AC_MSG_RESULT([no])
         ])
 
-    AC_LANG_RESTORE
-    LDFLAGS="$LDFLAGS_SAVE"
-    CFLAGS="$CFLAGS_SAVE"
-    CXXFLAGS="$CXXFLAGS_SAVE"
+   AC_LANG_RESTORE
 fi
 
 ########## INOTIFY TOOLS
